src/components/QuickGrid/DataSelectors.ts

```diff
@@ -64,19 +75,19 @@ const sortRows = (rows: Array<any>, sortColumnName: string,
     return rows;
 };
 
-const filterRows = (rows : Array<any>, columns: Array<GridColumn>, searchText : string) => {
+const filterRows = (rows: Array<any>, columns: Array<GridColumn>, searchText: string) => {
     if (!searchText) {
         return rows;
     }
 
     let filterText = searchText.toLowerCase();
 
-    let members = columns
-        .map(col => col.valueMember);
-    const newRows = rows.filter(row => {      
+    let members = columns.map(col => col.valueMember);
+
+    const newRows = rows.filter(row => {
         let visible = false;
         for (let value of members) {
-            if (row[value].toString().toLowerCase().includes(searchText)) {
```

Comment:
Why did you replace includes with indexOf?

Reply:
indexOf is just older function, its just so that both of grids have the same function